### PR TITLE
Added my own keymap for the planck light!

### PR DESCRIPTION
**Pull request by PickleJr**
Title pretty much says it all. I just added my own keymap.

### CODE REVIEW

File: keyboards/planck/keymaps/pickle_jr/keymap.c [hunk 1]
```diff
@@ -0,0 +1,196 @@
+#include "planck.h"
```
Comment (drashna) [inline keyboards/planck/keymaps/pickle_jr/keymap.c]: Could you replace this with `#include QMK_KEYBOARD_H`?

File: keyboards/planck/keymaps/pickle_jr/keymap.c [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+ * | Tab/CALC |   Q      |   W   |   E  |   R  |   T  |   Y  |   U  |   I  |   O   |   P      |  "/CALC |
+ * |----------+----------+-------+------+------+-------------+------+------+-------+----------+---------|
+ * | Esc      |   A      |   S   |   D  |   F  |   G  |   H  |   J  |   K  |   L   |   ;/Nav  | Enter   |
+ * |----------+----------+-------+------+------+------|------+------+------+-------+----------+---------|
+ * |   (/SFT  |   Z      |   X   |   C  |   V  |   B  |   N  |   M  |   ,  |   .   |   /      |  )/SFT  |
+ * |----------+----------+-------+------+------+------+------+------+------+-------+----------+---------|
+ * |   [/SPC  |   CTRL   |PrintS | GUI  | Bksp |Lower |Raise |  SPC | ALT  |VIMSAVE|  CTRL    |  ]/SPC  |
+ * `----------------------------------------------------------------------------------------------------'
+ */
+[_QWERTY] = {
```
Comment (drashna) [inline keyboards/planck/keymaps/pickle_jr/keymap.c]: Could you reformat these to use this formatting:
https://github.com/qmk/qmk_firmware/blob/master/keyboards/planck/keymaps/default/keymap.c#L58

File: keyboards/planck/keymaps/pickle_jr/config.h [hunk 3]
```diff
@@ -0,0 +1,40 @@
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/planck/keymaps/pickle_jr/config.h]: The first four lines (up to `#include "config_common.h"`) could be replaced with `#pragma once`, and the `#endif` at the end removed.

**Comment by PickleJr** [comment]
Updated!

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**